protocol/statetracker/consumer_state_tracker.go

```diff
@@ -48,6 +48,13 @@ func (cst *ConsumerStateTracker) RegisterConsumerSessionManagerForPairingUpdates
 	if !ok {
 		utils.LavaFormatFatal("invalid updater type returned from RegisterForUpdates", nil, utils.Attribute{Key: "updater", Value: pairingUpdaterRaw})
 	}
+
+	// register for updates in case of emergency mode is enabled
+	pairingUpdaterWithEmergencyRaw := cst.StateTracker.RegisterForEmergencyModeUpdates(ctx, pairingUpdater)
```

Comment:
don't do this code between the casting to pairing updater and pairingUpdater.RegisterPairing, making it complex and weird, do both registers without affecting one another